lib/util/support.js

```diff
@@ -34,6 +34,13 @@ function isSupported() {
     return false;
   }
 
+  // NOTE (joma): Return if iOS chrome version is not 14.3 and above.
+  if (isChromeIOS()) {
+    if (!isChromeVersionSupported()) {
+      return false;
+    }
+  }
```

Comment:
@PikaJoyce Do we need this check?

Reply:
Went to double check, it seems that we don't! Thanks Manjesh!